utils/commonActions.js

```diff
@@ -0,0 +1,32 @@
+const homePage = require("../test/pageObjects/homePage");
+const menSection = require("../test/pageObjects/menSectionPage");
+const productPage = require("../test/pageObjects/productPage");
+const signUpObjects = require("../test/pageObjects/signUpPage");
+
+async function signUp(firstName, lastName, email, password) {
+  await signUpObjects.firstNameInput.setValue(firstName);
+  await signUpObjects.lastNameInput.setValue(lastName);
+  await signUpObjects.signUpForNewsletterCheckbox.click();
+  await signUpObjects.emailInput.setValue(email);
+  await signUpObjects.passwordInput.setValue(password);
+  await signUpObjects.passwordConfirmationInput.setValue(password);
+  await signUpObjects.signUpButton.click();
+}
+
+const selectItem = async () => {
```

Comment:
This method should be more generic, since it is in commonActions and the plan is that it is reusable so it should at least receive parameters and select the item that you specified. I would advise you to create a method that search through the search bar and selects that Item.